utils/_context/containers.py

```diff
@@ -274,7 +274,8 @@ def _init_from_attrs(self, attrs):
 
         for var in attrs["Config"]["Env"]:
             key, value = var.split("=", 1)
-            self.env[key] = value
+            if value:
```

Comment:
Why this change ? 

I mean, I prefer to stick to what is exposed in the image.

Reply:
We are adding empty values for env key.
My point of view is that if we are not setting any value, better not to set this variable.
With this change, we avoid this error:

`(venv) roberto.montero@COMP-WMH2MQLQFT system-tests % ./run.sh OTEL_INTEGRATIONS                            
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/_pytest/main.py", line 264, in wrap_session
INTERNALERROR>     config._do_configure()
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/_pytest/config/__init__.py", line 996, in _do_configure
INTERNALERROR>     self.hook.pytest_configure.call_historic(kwargs=dict(config=self))
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/pluggy/_hooks.py", line 277, in call_historic
INTERNALERROR>     res = self._hookexec(self.name, self.get_hookimpls(), kwargs, False)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/pluggy/_manager.py", line 80, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 60, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/pluggy/_result.py", line 60, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/pluggy/_callers.py", line 39, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/conftest.py", line 53, in pytest_configure
INTERNALERROR>     context.scenario.configure(config.option)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/utils/_context/_scenarios.py", line 591, in configure
INTERNALERROR>     super().configure(option)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/utils/_context/_scenarios.py", line 256, in configure
INTERNALERROR>     container.configure(self.replay)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/utils/_context/containers.py", line 424, in configure
INTERNALERROR>     self.appsec_rules_version = Version(appsec_rules_version, "appsec_rules")
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/utils/_context/library_version.py", line 145, in __init__
INTERNALERROR>     super().__init__(version)
INTERNALERROR>   File "/Users/roberto.montero/Documents/development/system-tests/venv/lib/python3.9/site-packages/packaging/version.py", line 266, in __init__
INTERNALERROR>     raise InvalidVersion(f"Invalid version: '{version}'")
INTERNALERROR> packaging.version.InvalidVersion: Invalid version: ''`